dat/data.base

```diff
@@ -1234,6 +1293,23 @@ major demon
 	them difficult to deal with, even on the rare occasions when
 	they are friendly.
 	  [ The Tough Guide to Fantasyland, by Diana Wynne Jones ]
+demonbane
```

Comment:
This seems like another "has no bearing on the item/monster other than an abstract association with its name" one to me.  The apologetics is not totally unjustified since it's the sac gift for Priests, so I guess that is a secondary link, but there must be other options in that field that are actually referring to a (metaphorical) weapon driving out demons rather than "the continued faith of a man confronted with a seemingly godless universe".  E.g. Matthew 10:34 could be appropriate, if maybe a little obvious.